docs/src/pathing.md

```diff
@@ -36,6 +36,7 @@ dfs_tree
 maximum_adjacency_visit
 bfs_parents
 has_path
+get_path
```

Comment:
bikeshed: should we call this `dfs_get_path`?

Reply:
I have no opinion about the names. Bikeshed away.

Reply:
For bikeshedding it might also be worth considering switching from `get` to `find`. And possibly from `path` to `simple_path`.